### PR TITLE
French translation for the `ascent-override` CSS descriptor reference.

### PR DESCRIPTION
**Pull request by audrasjb**
This PR adds a French translation for the `ascent-override` CSS descriptor reference.

Please note that I asked a question about the best practice for `cssxref` references translation here: https://matrix.to/#/!pGdhkeiUTYDIWuskqX:mozilla.org/$bdvXKWkU1RyybuAjp0JmacC9RK8I2r7sF6yuhtIIFZM?via=mozilla.org&via=t2bot.io&via=matrix.org

### CODE REVIEW

File: files/fr/web/css/@font-face/ascent-override/index.html [hunk 1]
```diff
@@ -0,0 +1,74 @@
+---
+title: ascent-override
+slug: Web/CSS/@font-face/ascent-override
+browser-compat: css.at-rules.font-face.ascent-override
+translation_of: 'Web/CSS/@font-face/ascent-override'
+---
+<p>{{CSSRef}}</p>
+
+<p>Le descripteur CSS <strong><code>ascent-override</code></strong> définit la mesure du jambage supérieur (ascendant) de la police. La mesure du jambage supérieur correspond à la distance verticale par rapport à la hauteur d'X (hauteur du caractère bas de casse, à l'exclusion des jambages inférieurs et supérieurs) que CSS doit utiliser pour agencer les boîtes dans un contexte de formatage en ligne.</p>
```
Comment (SphinxKnight) [inline files/fr/web/css/@font-face/ascent-override/index.html:9]: I know it's a bit of a stretch here but I'd point to https://en.wikipedia.org/wiki/Ascender_(typography) to have a visual clue of an ascender (/me crying a bit as the wiki page for "Hauteur d'ascendante" in fr doesn't exist yet)

File: files/fr/web/css/@font-face/ascent-override/index.html [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+<pre class="brush: css">ascent-override: normal;
+ascent-override: 90%;</pre>
+
+<h3 id="values">Valeurs</h3>
+
+<dl>
+  <dt><code>normal</code></dt>
+  <dd>La valeur par défaut. La mesure du jambage supérieur est obtenue à partir du fichier de police.</dd>
+  <dt><code>&lt;percentage&gt;</code></dt>
+  <dd>Une valeur exprimée en {{cssxref("&lt;percentage&gt;", "&lt;pourcentage&gt;")}}.</dd>
```
Comment (SphinxKnight) [inline files/fr/web/css/@font-face/ascent-override/index.html:22]: As discussed on Matrix, I think it's better to leave cssxref calls for CSS types in English.
However, that should not prevent us from writing stuff like "Une valeur exprimée en pourcentage (type {{cssxref("&amp;lt;percentage&amp;gt;)}} qui traduit l'augmentation ou la réduction par rapport à la taille de la police ({{cssxref("font-size")}}."

I find the English version a bit dry here as there is no baseline for what the percentage is relative too so I'd include a few additional explanations. I'll leave it up to you if you want to fix upstream  ^- ^

**Review by SphinxKnight — CHANGES_REQUESTED** [review]
Slight changes and we're good to merge :)

File: files/fr/web/css/@font-face/ascent-override/index.html (hunk 1 above)
Comment (audrasjb) [reply files/fr/web/css/@font-face/ascent-override/index.html:9]: Should we link the "Jambage" Wikipedia article instead?
https://fr.wikipedia.org/wiki/Jambage
(it includes both "jambage inférieur" and "jambage supérieur")

File: files/fr/web/css/@font-face/ascent-override/index.html (hunk 2 above)
Comment (audrasjb) [reply files/fr/web/css/@font-face/ascent-override/index.html]: Fixed in https://github.com/mdn/translated-content/pull/1752/commits/600211688c290bd6c766b2a1f9b7bf8f505cd3cf and https://github.com/mdn/translated-content/pull/1752/commits/3be2f814cbedad8a12671dce7e74be7533b451cd

File: files/fr/web/css/@font-face/ascent-override/index.html (hunk 1 above)
Comment (audrasjb) [reply files/fr/web/css/@font-face/ascent-override/index.html]: I committed a proposal: https://github.com/mdn/translated-content/pull/1752/commits/2449a8a4f2b810a3d324962d5b115ec07840f71d
waiting for your review (no hurry!) before applying it to the other occurences (like descent-override)

**Review by SphinxKnight — APPROVED** [review]
Looks good to me :) (I should maybe start contributing to Wikipédia :) )

**Pull request merged**